llvm/lib/WindowsManifest/WindowsManifestMerger.cpp: xmlKeepBlanksDefault is deprecated API

Compiling LLVM 18.1.2 results in the following:

```
[247/3797] Building CXX object lib/WindowsManifest/CMakeFiles/LLVMWindowsManifest.dir/WindowsManifestMerger.cpp.o
/tera/tera/debo/Projects/llvm/llvmorg-18.1.2/llvm/lib/WindowsManifest/WindowsManifestMerger.cpp:672:5: warning: 'xmlKeepBlanksDefault' is deprecated [-Wdeprecated-declarations]
  672 |     xmlKeepBlanksDefault(0);
      |     ^
/usr/local/include/libxml2/libxml/parser.h:956:1: note: 'xmlKeepBlanksDefault' has been explicitly marked deprecated here
  956 | XML_DEPRECATED XMLPUBFUN int
      | ^
/usr/local/include/libxml2/libxml/xmlversion.h:447:43: note: expanded from macro 'XML_DEPRECATED'
  447 | #    define XML_DEPRECATED __attribute__((deprecated))
      |                                           ^
```

Note that I am using `libxml2-2.12.6`.

--- a/llvm/lib/WindowsManifest/WindowsManifestMerger.cpp
+++ b/llvm/lib/WindowsManifest/WindowsManifestMerger.cpp
@@ -669,7 +669,6 @@ WindowsManifestMerger::WindowsManifestMergerImpl::getMergedManifest() {
     xmlDocSetRootElement(OutputDoc.get(), CombinedRoot);
     assert(nullptr == xmlDocGetRootElement(CombinedDoc));
 
-    xmlKeepBlanksDefault(0);
     xmlChar *Buff = nullptr;
     xmlDocDumpFormatMemoryEnc(OutputDoc.get(), &Buff, &BufferSize, "UTF-8", 1);
     Buffer.reset(Buff);

